File: plugins/devkit/devkit-java-tests/testData/inspections/useJBColorFix/UseJBColorConstantInMethodParam.java
import java.awt.Color;

class UseJBColorConstructor {
  void any() {
    takeColor(<warning descr="'java.awt.Color' used instead of 'JBColor'">Color.g<caret>reen</warning>);
  }

  void takeColor(Color color) {
    // do nothing
  }
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useJBColorFix/UseJBColorConstantInMethodParam.kt
import java.awt.Color

class UseJBColorConstantInMethodParam {
  fun any() {
    takeColor(<warning descr="'java.awt.Color' used instead of 'JBColor'">Color.g<caret>reen</warning>)
  }

  @Suppress("UNUSED_PARAMETER")
  fun takeColor(color: Color) {
    // do nothing
  }
}
